aws/src/main/java/org/apache/iceberg/aws/glue/GlueCatalog.java

```diff
@@ -524,7 +528,11 @@ public Map<String, String> loadNamespaceMetadata(Namespace namespace)
 
   @Override
   public boolean dropNamespace(Namespace namespace) throws NamespaceNotEmptyException {
-    namespaceExists(namespace);
+    // check if namespace exists
+    if (!namespaceExists(namespace)) {
+      LOG.error("Cannot drop namespace {} because it does not exist.", namespace);
```

Comment:
I don't think there needs to be an error log (or any log for this case) at the catalog level. I think different integrations can use the boolean response and make their own judgement on how to handle the drop not occurring due to the namespace not existing (which may be logging, throwing something themselves, etc).

Reply:
Followed the similar pattern of logging from dropTable where we log the error in case `EntityNotFoundException` and return false. I agree that it is redundant, will go ahead and remove the log.